tests/integration/replica-tunnel.tcl

```diff
@@ -0,0 +1,79 @@
+start_server {tags {needs:repl external:skip "quit"}} {
```

Comment:
A few more that would be good:

1. Node A FAILOVER TO node B TUNNEL followed by Node B FAILOVER TO node A TUNNEL (while keeping tunnels alive)
2. Tunneled connection is authenticated to a user